network/simulation/simulation.go

```diff
@@ -107,6 +107,7 @@ func NewBzzInProc(services map[string]ServiceFunc) (s *Simulation) {
 		hp := network.NewHiveParams()
 		hp.KeepAliveInterval = time.Duration(200) * time.Millisecond
 		hp.Discovery = false
+		hp.DisableAutoConnect = true
```

Comment:
this is incorrect here, but All snapshot tests need to set this to true. So this param needs to be passed down in node config.